main.js

```diff
@@ -43,7 +43,7 @@ playground_main :- QueryStr = "${queryStr}", read_term_from_chars(QueryStr, Quer
 		historyOutput.textContent = "false.";
 	    }
 	    element.appendChild(historyOutput);
-	    history.appendChild(element);
+	    history.prepend(element);
```

Comment:
This reverses the order. Originally it was like this, but it was also not liked (see https://github.com/aarroyoc/scryer-playground/issues/3). I think we should maintain the current order, just showing the latest at top. Probably needs more UI changes

Reply:
ok, another attempt: i've kept the order the same, but i've added a div under the query box to show the result of the latest query.